tests/std/tests/P0769R2_shift_left_shift_right/test.cpp

```diff
@@ -7,6 +7,12 @@
 #include <iterator>
 #include <list>
 #include <vector>
+#include <version>
+
+#if __cpp_lib_shift >= 202202L
+// Test ranges::shift_left, ranges::shift_right from P2440R1
+#define TEST_RANGES
+#endif

```

Comment:
I think we comment out `#endif`

So it should be 

`#endif // __cpp_lib_shift >= 202202L`